src/renderer/src/components/TransactionActivityList/TransactionActivityListEvent.tsx

```diff
@@ -0,0 +1,159 @@
+  address: string
+  addressName?: string
+}
+
+type TProps = {
+  event: TFullTransactionEvent
+}
+
+const ColumnContent = ({ data }: TColumnContentProps) =>
+  typeof data === 'string' || typeof data === 'number' ? (
+    <TransactionActivityListTooltip data={data}>
+      <span className="inline-block max-w-20 truncate">{data}</span>
+    </TransactionActivityListTooltip>
+  ) : (
+    data
+  )
+
+const Column = ({ data, label, url }: TColumnProps) => (
+  <div className="flex flex-col">
+    {label && <p className="font-medium text-gray-300">{label}</p>}
+
+    {url ? (
+      <Link to={url} target="_blank" className="flex text-neon">
+        <ColumnContent data={data} />
+      </Link>
+    ) : (
+      <span className="flex text-white">
+        <ColumnContent data={data} />
+      </span>
+    )}
+  </div>
+)
+
+const AddressColumnData = ({ address, addressName }: TAddressColumnDataProps) => {
+  const tooltipLabel = `${address} ${addressName ? `(${addressName})` : ''}`.trim()
+  const textLabel = StringHelper.truncateString(addressName ? addressName : address, 10)
+
+  return (
+    <TransactionActivityListTooltip data={tooltipLabel}>
+      <span>{textLabel}</span>
+    </TransactionActivityListTooltip>
+  )
+}
+
+export const TransactionActivityListEvent = ({ event }: TProps) => {
+  const { t } = useTranslation('components', { keyPrefix: 'transactionActivityList.event' })
+
+  const { eventType, amount, methodName, to, toUrl, toAccount, from, fromUrl, fromAccount, hash, hashUrl, tokenType } =
+    event
+
+  const tokenTypeLabel = !!tokenType && tokenType !== 'generic' ? tokenType.toUpperCase() : ''
+  const methodNameLabel = `${methodName} ${tokenTypeLabel ? `(${tokenTypeLabel})` : ''}`.trim()
+  const toName = toAccount?.name
+  const fromName = fromAccount?.name
+
+  return (
+    <div className="flex-grow flex justify-between items-center gap-x-2 ml-20 pl-4 pr-2 h-[49px] min-h-[49px] max-h-[49px] whitespace-nowrap overflow-x-auto overflow-y-hidden">
+      {!!hash && <Column label={t('columns.hashLabel')} data={hash} url={hashUrl} />}
+
+      {!!methodNameLabel && (
+        <Column
+          label={t('columns.methodNameLabel')}
+          data={
+            <TransactionActivityListTooltip data={methodNameLabel} className="uppercase">
+              <span className="inline-block uppercase truncate max-w-32">{methodNameLabel}</span>
+            </TransactionActivityListTooltip>
+          }
+        />
+      )}
+
+      {!!from && (
+        <Column
+          label={t('columns.fromLabel')}
+          data={<AddressColumnData address={from} addressName={fromName} />}
+          url={fromUrl}
+        />
+      )}
+
+      {!!to && (
+        <Column
+          label={t('columns.toLabel')}
+          data={<AddressColumnData address={to} addressName={toName} />}
+          url={toUrl}
+        />
+      )}
+
+      {!!amount && amount !== '0' && <Column label={t('columns.amountLabel')} data={amount} />}
```

Comment:
It would be better to transform the amount to number and then check if it is greater than 0, because the string could be "00" or "0.0".

```suggestion
      {!!amount && Number(amount) > 0 && <Column label={t('columns.amountLabel')} data={amount} />}
```